oak_attestation_verification/src/claims.rs

```diff
@@ -83,6 +83,10 @@ pub struct ClaimPredicate<S> {
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(rename = "claimSpec")]
     pub claim_spec: Option<S>,
+    // Whether the binary or an attachment is endorsed. If it is the
```

Comment:
This comment does not align with my expectation of how the `usage` field will be used. The usage field should replace the subject name to indicate which field in the evidence the endorsement targets to avoid using a wrong endorsement for an evidence field. That implies whether an attachment is expected and how it is parsed, but the main point is to be explicit about which piece of the evidence is targeted by the endorsement. Over time we must check that we have the correct expected `usage` for every field in the evidence.

Reply:
Thanks, updated the comment to shift focus.